integrationtest/TargetProjects/Library/Library.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6</TargetFramework>
```

Comment:
How do we guarantee backwards compatibility if we no longer test dotnet core

Reply:
Good point. I'll see if we can keep some on 3.1